### PR TITLE
Plotting page update

### PR DESCRIPTION
**Pull request by birdage**
DO NOT MERGE YET!!!!!!!

### CODE REVIEW

**Comment by birdage** [comment]
@DanielJMaher @oceanzus this removes the bad commit, but please verify it....

**Comment by birdage** [comment]
still working on it

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 *.swp
+*.swo
```
Comment (DanielJMaher) [inline .gitignore]: @birdage Is this something we need/want.  This is the only remnant from Dan's PR

Comment (birdage) [inline .gitignore]: @DanielJMaher i think it will be fine.....

**Comment by birdage** [comment]
@DanielJMaher DONT merge yet tho

File: .gitignore (hunk 1 above)
Comment (DanielJMaher) [inline .gitignore]: As long as you knew it was there.

Comment (birdage) [inline .gitignore]: yeah i saw it, i rebased from origin master so i think were ok

**Pull request merged**